src/app.global.scss

```
@@ -43,24 +43,25 @@ a {
 }
 // style for links that are menu items or buttons (not just text)
 .gc360_link {
-  color: $neutral-black;
+  color: var(--mui-palette-neutral-contrastText);
+  background-color: var(--mui-palette-neutral-main);
   &:hover {
-    color: $neutral-black !important;
+    color: var(--mui-palette-neutral-contrastText) !important;
     text-decoration: none !important;
   }
 }
 // style for links that are simply text and therefore need more of a sign that they are clickable
 .gc360_text_link {
-  color: $primary-blue;
+  color: var(--mui-palette-primary-main);
   transition: color 200ms;
   &:hover {
-    color: $primary-cyan !important;
+    color: var(--mui-palette-secondary-main) !important;
   }
 }
 
 @media (max-width: $break-md) {
   .gc360_a_link {
-    color: $primary-cyan !important;
+    color: var(--mui-palette-secondary-train) !important;
```

Is this supposed to be `main` instead of `train`?
```suggestion
    color: var(--mui-palette-secondary-main) !important;
```

---

Please commit the suggestion.